app/models/user_action.rb

```diff
@@ -290,17 +287,18 @@ def self.log_action!(hash)
 
         update_like_count(user_id, hash[:action_type], 1) if topic && !topic.private_message?
 
+        user_ids = user_id != action.acting_user_id ? [user_id] : nil
```

Comment:
FIX: 3/3 - we'll only publish the user_action when the `user` is different from the `acting_user`